README.md

```diff
@@ -258,6 +258,8 @@ List of useful, silly and [awesome](#awesome-) lists curated on GitHub. Contribu
 * [Facets](https://github.com/O-I/Facets) – One-liners in Ruby
 * [fks](https://github.com/JacksonTian/fks) _In Chinese_ – Frontend Knowledge Structure.
 * [flat-file-cms](https://github.com/ahadb/flat-file-cms) – Stictly flat-file cms systems.
+* [FMHY](https://github.com/fmhy/FMHY) – The largest collection of free stuff on the internet.
```

Comment:
suggestion: link to `fmhy/edit` (the naming here per guidelines is tricky, so let's keep `fmhy` in the name)

```suggestion
* [fmhy/edit](https://github.com/fmhy/edit) – The largest collection of free stuff on the internet.
```